core/app/models/spree/stock_item.rb

```diff
@@ -6,6 +6,7 @@ class StockItem < Spree::Base
     if defined?(Spree::Webhooks)
       include Spree::Webhooks::HasWebhooks
     end
+    include Spree::Metadata
```

Comment:
@aleksandarpetrushev this is already done on line 5

Reply:
I missed that, sorry